### PR TITLE
Grids: Update

### PR DESCRIPTION
**Pull request by imincheva**
- Display density has been replaced with size where necessary (including file name, links and images names)
- All images in the Grids pages containing export button have been updated to reflect the visual design changes

### CODE REVIEW

File: en/images/grid_excel_style_filter_size.png
Comment (yradoeva) [inline en/images/grid_excel_style_filter_size.png:1]: As far as I see, the Excel style filter has visual changes - the searchbar and the dashed line are different, the roundness of the dialog.

**Comment by yradoeva** [comment]
There are some inaccuracies I've noticed (still checking and might find more):

**Grid Column Pinning**
then set the feature property to Pinned in the properties par


**Grid Column Hiding** 

Toolbar and Header Cell ∞


**Grid Column Moving** 

unknown


**Grid Column Resizing**
the column as resizable as far as code generation is concerned
.
The Grid has a Colum Resizing Indicator symbolcomponent under Features that represents a

**Comment by yradoeva** [comment]
**Grid Advanced Filter**
The dialog has some visual changes - the background of the header and the body, the dashed line, the roundness. Changes affect all images on the page.

File: en/patterns/grid.md
Comment (yradoeva) [inline en/patterns/grid.md:1]: (row 39) Should we change Display Density to Size, or remove it at all?

Comment (imincheva) [reply en/patterns/grid.md:1]: we should remove it, since we've removed it from the pattern's name

**Comment by yradoeva** [comment]
**Patterns**

**Grid, Tree Grid** - images - there is a slight difference in the Paginator

File: en/patterns/grid.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The Grid Pattern with Column Moving, Summaries and Paging is shown below:
 
 <img class="responsive-img" src="../images/grid.png" srcset="../images/grid@2x.png 2x" />
 
-List of all the Grid functionality:
+List of all the Grid functionalities:
```
Comment (yradoeva) [inline en/patterns/grid.md:15]: @sdimchevski Should we align this functionalities list in the Grid / Hierarchical Grid / Tree Grid Pattern pages? Currently on the Grid and Hierarchical Grid pages it looks the same way, and on the Tree Grid page we have slight difference + links.

**Review by yradoeva — APPROVED** [review]

**Pull request merged**